### PR TITLE
Added UPI

### PR DESCRIPTION
**Pull request by sidharthv96**

### CODE REVIEW

**Comment by biswaz** [comment]
direct link to cmdrf is better IMHO

**Comment by kira0204** [comment]
Fixes #25

**Comment by sidharthv96** [comment]
One click payment is possible here right?

File: mainapp/templates/mainapp/contributor_form.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
       Branch: City Branch, TVM<br>
       IFS Code: SBIN0070028<br>
       <br>
-      UPI 1: cmdrf@fbl<br>
-      UPI 2: keralacmdrf@sib<br>
+      UPI 1: <a href="upi://pay?pa=cmdrf@fbl&pn=Chief%20Minister%27s%20Distress%20Relief%20Fund">cmdrf@fbl</a><br>
+      UPI 2: <a href="upi://pay?pa=keralacmdrf@sib&pn=Chief%20Minister%27s%20Distress%20Relief%20Fund">keralacmdrf@sib</a><br>
```
Comment (akhilerm) [inline mainapp/templates/mainapp/contributor_form.html]: Please add `keralacmdrf@sbi` also.

**Comment by SpEcHiDe** [comment]
I think hotlinking UPI:// protocol will lead to unsupported protocol in some browsers like Firefox.

File: mainapp/templates/mainapp/contributor_form.html (hunk 1 above)
Comment (sidharthv96) [reply mainapp/templates/mainapp/contributor_form.html]: Isn't it better to keep the options short. It might lead to extra confusion for the users.

Comment (josethomasd) [reply mainapp/templates/mainapp/contributor_form.html]: Only keralacmdrf@sbi is working with UPI apps. Rest of them are getting detected as invalid UPI id (Tested with Tez, BHIM, PhonePe & PayTM) .

Comment (josethomasd) [reply mainapp/templates/mainapp/contributor_form.html]: Links are working with Chrome and is opening on UPI apps correctly.

Comment (gauravsaini) [reply mainapp/templates/mainapp/contributor_form.html]: Can somebody also add the consolidated qr codes as image to the repo ? That can then be referred to later on.

**Comment by jilvin** [comment]
Please checkout #510.

**Comment by remedcu** [comment]
Related PR's & Issues : #36 #52 #90 #141 #350 #402 #468 #510 #614 

Quoting @biswaz 

"Officials want just a link to the official portal. Sorry KSITM requirements are our first priority. I know exactly what it feels like be on the contributor side. Been there, done that."

So, I believe this can be closed.

@syam3526 @vigneshhari @vrthra @bobinson

**Pull request closed without merging**